settlement/tests/treasury.rs

```diff
@@ -177,7 +177,7 @@ fn relay_1() {
 
     let agenda = Agenda {
         height: 1,
-        author: keys[0].0.clone(), // Note that keys[0] is member-0001
+        author: reserved_state.query_name(&keys[0].0).unwrap(), // Note that keys[0] is member-0001
```

Comment:
Delete the comment please, it's outdated.

Reply:
fixed.